python_legacy/iceberg/hive/hive_tables.py

```diff
@@ -67,9 +69,11 @@ def new_table_ops(self, conf, database, table):
 
     def get_client(self) -> HMSClient:
         from urllib.parse import urlparse
+        iprot = self.conf.get(HiveTables.IPROT)
```

Comment:
Do we need to put a default value? Will it still work if iprot and oprot are None value?

Reply:
@jun-he I don't think we need a default value. As iprot is None, it will just use host and port value. https://github.com/gglanzani/hmsclient/blob/master/hmsclient/hmsclient.py#L42